src/logo/logo.js

```diff
@@ -3,23 +3,23 @@
 import logo from '../models/logo-flat.json';
 import particle from '../textures/particle.png';
 
-import THREE from 'three';
+import * as THREE from 'three';
```

Comment:
perhaps not, I think we can do some refactoring in another branch later, this one was intended only to resurrect the project and fix a few bugs. Let's do on thing a time, yeah. :)